connman/src/main.c

```diff
@@ -118,6 +120,7 @@ static struct {
 	.vendor_class_id = NULL,
 	.fallback_device_types = NULL,
 	.enable_login_manager = false,
+	.localtime = DEFAULT_LOCALTIME,
```

Comment:
Note: this assigns a `const char *` to a `char *`.

Reply:
Yep, modified the use of this to be in line with the status check urls with 481c63fe79c452579491f83ab27753ac3f6d4224 so the default value is returned in the getter when `localtime` is `NULL`.